acj/static/modules/common/avatar-directive.js

```diff
@@ -6,12 +6,13 @@
 			scope: {
 				userId: '=',
 				avatar: '=',
+                forceDefault: '=?',
 				displayName: '=?',
 				fullName: '=?',
 				me: '=?'
 			},
 			template: '<a ng-href="#/user/{{ userId }}">' +
-						'<img src="//www.gravatar.com/avatar/{{ avatar }}?s=32&d=retro" /> ' +
+						'<img src="//www.gravatar.com/avatar/{{ avatar }}?s=32&d=retro{{ forceDefault ? \'&f=y\' : \'\' }}" /> ' +
```

Comment:
Yes you are correct, its still possible to get around this.

We could use the userid + some string for the hash so that the avatars are still distinct from each other.